packages/portal-proto/src/components/SwitchSpring/shared/SwitchSpring.tsx

```diff
@@ -55,36 +39,27 @@ const SwitchSpring: React.FC<SwitchSpringProps> = ({
       disabled={!tooltip}
       data-testid="tooltipSwitchSpring"
     >
-      <animated.div
-        data-testid="button-middle-switchSpring"
-        style={sliderSpring}
-        className={classNames(
-          "border border-lightgray",
-          disabled ? "cursor-not-allowed" : "cursor-pointer",
-          {
-            "rounded-xl": icon,
-          },
-          isActive ? "bg-activeColor" : "bg-gray-300",
-        )}
-        onClick={toggleSwitch}
-        onKeyDown={createKeyboardAccessibleFunction(toggleSwitch)}
-        aria-disabled={disabled}
-        tabIndex={0}
-        aria-label={tooltip}
-      >
-        <animated.div
-          data-testid="button-bottom-switchSpring"
-          style={ballSpring}
-          className={`ml-1 border-2 rounded-sm ${
-            disabled ? "border-gray-300" : "border-activeColor"
-          }
-             bg-white ${icon && `rounded-xl`} ${
-            isActive && `bg-lightgray`
-          } h-5`}
-        >
-          <div className={margin}>{icon}</div>
-        </animated.div>
-      </animated.div>
+      {/* div added otherwise tooltip does not work in mantine 6 */}
+      <div>
+        <Switch
+          data-testid="switchSpring-checkbox"
```

Comment:
Can you call this "button-switchspring" instead?

Reply:
yes we can, just FYI the label is applied to the checkbox when the component is rendered not to the button, would you still like me to change it?